src/d3d9/d3d9_surface.h

```diff
@@ -20,8 +20,8 @@ namespace dxvk {
     D3D9Surface(
             D3D9DeviceEx*             pDevice,
       const D3D9_COMMON_TEXTURE_DESC* pDesc,
-            IUnknown*                 pContainer,
-            HANDLE*                   pSharedHandle);
+            IUnknown*                 pContainer    = nullptr,
+            HANDLE*                   pSharedHandle = nullptr);
```

Comment:
Not a huge fan of default parameters where  we can avoid them, can we just make a separate constructor instead that just forwards to this one?